Ginger/Ginger/Actions/ActionEditPages/ActOcrEditPage.xaml.cs

```diff
@@ -229,5 +230,18 @@ private void SelectColumnValueRdb_Checked(object sender, RoutedEventArgs e)
             xColumnWhereValue.IsEnabled = !rb.IsChecked.Value;
             xOperationCombo.IsEnabled = !rb.IsChecked.Value;
         }
+
+        private void PdfPassword_LostKeyboardFocus(object sender, KeyboardFocusChangedEventArgs e)
+        {
+            UCValueExpression uv = (UCValueExpression)sender;
+            TextBox pdfPasswordTextBox = uv.ValueTextBox;
```

Comment:
Why we need TextBox object here?

Reply:
![Screenshot (37)](https://github.com/Ginger-Automation/Ginger/assets/58476741/6a50431a-ca89-43e6-99d2-62ca64bc77aa)

In The mentioned Image. We have UCValueExpression in XAML to input password. As UCValueExpression doesn't provide with a property that returns the text Entered in it. Therefore,  converted the UCValueExpression to TextBox.